lib/Elastica/Index/Settings.php

```diff
@@ -185,13 +194,12 @@ public function setBlocksWrite($state = true)
      */
     public function getBlocksMetadata()
     {
-        // TODO will have to be replace by block.metadata.write once https://github.com/elasticsearch/elasticsearch/pull/9203 has been fixed
```

Comment:
This has been merged and makes it possible to use the correct `read_only` setting again which was changed in #738